client-data/tools/zoom/zoom.js

```diff
@@ -82,14 +82,22 @@
     function onwheel(evt) {
         evt.preventDefault();
         if (evt.ctrlKey || Tools.curTool === zoomTool) {
+            // zoom
             var scale = Tools.getScale();
             var x = evt.pageX / scale;
             var y = evt.pageY / scale;
             setOrigin(x, y, evt, false);
             animate((1 - ((evt.deltaY > 0) - (evt.deltaY < 0)) * 0.25) * Tools.getScale());
+        } else if (evt.altKey) {
+            // make finer changes if shift is being held
+            var change = evt.shiftKey ? 1 : 5;
+            // change tool size
+            Tools.setSize(Tools.getSize() - ((evt.deltaY > 0) - (evt.deltaY < 0)) * change);
```

Comment:
Couldn't we make it directly proportional to the deltaY ?

Reply:
That would be different in each browser (same issue that the scroll to zoom function had initially). Firefox reports a delta of 3 whereas chrome reports 53 (at least on my device).